resources/js/Components/Event.svelte

```diff
@@ -0,0 +1,48 @@
+<script>
+    import clsx from 'clsx';
+    import route from '@/route';
+    import { Link } from '@components/icons';
+    import { inertia } from '@inertiajs/svelte';
+    export let event;
+
+    const formattedDate = new Date(event.date).toLocaleDateString('en-US');
+    async function linkClicked() {
+        await navigator.clipboard.writeText(route('event.show', event));
```

Comment:
This doesn't work on pages without https apparently, we should look for a library or a workaround and move the function into a helper utility.